lib/shopify_cli/context.rb

```diff
@@ -656,17 +659,15 @@ def ctx_path(fname)
     def retrieve_latest_gem_version
       response = Net::HTTP.get_response(GEM_LATEST_URI)
       latest = JSON.parse(response.body)
-      latest["version"]
+      ShopifyCLI::Config.set(VERSION_CHECK_SECTION, LATEST_VERSION_FIELD, latest["version"])
     rescue
       nil
+    ensure
+      ShopifyCLI::Config.set(VERSION_CHECK_SECTION, LAST_CHECKED_AT_FIELD, Time.now.to_i)
```

Comment:
Still wait until tomorrow to try if something went wrong

Reply:
Something is wrong. This doesn't work as intended. Version check repeats on every command. See #2382